src/button.scss

```diff
@@ -261,6 +269,20 @@ a.#{$block} {
     }
   }
 
+  &--text-alignment {
+    &-left {
```

Comment:
We should make left as the default and only use the center or right as modifiers